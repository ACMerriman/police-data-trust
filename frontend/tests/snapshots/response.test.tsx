import * as React from 'react'
import renderer from "react-test-renderer"
import {
  RegistrationResponse,
  PassportApplicationResponse
} from '../../compositions/enrollment-response/enrollment-response'

it('renders Register:Success correctly', () => {
  const tree = renderer.create(<RegistrationResponse isSuccess />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders Register:Fail correctly', () => {
  const tree = renderer.create(<RegistrationResponse isSuccess={false} />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders Passport:Success correctly', () => {
  const tree = renderer.create(<PassportApplicationResponse isSuccess />).toJSON()
  expect(tree).toMatchSnapshot()
})

it('renders Passport:Fail correctly', () => {
  const tree = renderer.create(<PassportApplicationResponse isSuccess={false} />).toJSON()
  expect(tree).toMatchSnapshot()
})