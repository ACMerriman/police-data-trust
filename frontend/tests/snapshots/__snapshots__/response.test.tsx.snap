// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Passport:Fail correctly 1`] = `null`;

exports[`renders Passport:Success correctly 1`] = `null`;

exports[`renders Register:Fail correctly 1`] = `null`;

exports[`renders Register:Success correctly 1`] = `null`;
